<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaIngresoDatos</class>
 <widget class="QMainWindow" name="PantallaIngresoDatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpIngresoDatos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>941</width>
      <height>701</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV50</family>
     </font>
    </property>
    <property name="title">
     <string>Ingreso de Datos</string>
    </property>
    <widget class="QLabel" name="lblTiempoSimulacion">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tiempo de Simulación (días):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTiempoSimulacion">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpDistribuciones">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>881</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>Distribuciones</string>
     </property>
     <widget class="QLabel" name="lblFinAtencionConsulta">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Uniforme Capacidad Fin Trabajo</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblA">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblB">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>350</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>B:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUnifA">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUnifB">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>340</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Numero de trabajos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>de Frenos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNT5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNT4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNT3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNT2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNT1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Numero de</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNS5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNS2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNS3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Semanas</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNS4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNS1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtC4">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>180</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1600</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cant. Trabajos </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Costo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtC2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>120</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>950</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtC3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1300</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTA2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>120</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTA3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Acordados x Mes</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtC1">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>90</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>500</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTA4">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTA1">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>90</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Costo trabajo fuera de cantidad:  $</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtC5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>270</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>400</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>21</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>$</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>170</y>
        <width>21</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>$</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>140</y>
        <width>21</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>$</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>110</y>
        <width>21</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>$</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lblPrimeraFila">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>640</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fila inicial a mostrar: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPrimeraFila">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>630</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPorcNoCalm_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>670</y>
       <width>541</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Se mostraran las siguientes 400 filas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSimular">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>630</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Simular</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>711</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV50</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simulación Trabajo de Frenos</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
